packages/create-sitecore-jss/src/templates/nextjs-sxa/src/lib/page-props-factory/plugins/error-pages.ts

```diff
@@ -1,53 +0,0 @@
-import { GetServerSidePropsContext, GetStaticPropsContext } from 'next';
```

Comment:
I assume that we can still keep this plugin and provide all the required data as it was, isn't it?

Reply:
No, it's redundant now. We moved logic into 400/500.tsx pages